localstack/utils/common.py

```diff
@@ -1458,6 +1459,19 @@ def parse_json_or_yaml(markup: str) -> JsonComplexType:
                 raise
 
 
+def try_json(data: str):
+    """
+    Tries to deserialize json input to object if possible, otherwise returns original
+    :param data: string
+    :return: deserialize version of input
+    """
+    try:
+        return json.loads(to_str(data or "{}"))
+    except JSONDecodeError:
+        LOG.warning("failed serialize to json, fallback to original")
```

Comment:
Something that we should agree on for sure. I use warning to flag something went sideways but there is no reason to panic - also the message is pretty unique to the error and is a valid error and expected. All to avoid raising red flags.